modules/apps/callbacks/ibc_middleware.go

```diff
@@ -20,6 +20,7 @@ import (
 var (
 	_ porttypes.Middleware            = (*IBCMiddleware)(nil)
 	_ porttypes.PacketDataUnmarshaler = (*IBCMiddleware)(nil)
+	_ porttypes.UpgradableModule      = (*IBCMiddleware)(nil)
```

Comment:
I assume its intensional that we never added an app upgrade callback for `chanUpgradeConfirm`, its been so long that I don't really remember, but I think it was

Reply:
yes it is intentional. I think there's nothing to do in that step since it is just acknowledging that the counterparty set your timeout timer